tiledb/bioimg/converters/base.py

```diff
@@ -22,6 +20,14 @@
 import tiledb
 from tiledb.cc import WebpInputFormat
 
+from ..helpers import (
+    _create_image_pyramid,
+    _get_pixel_depth,
+    _get_schema,
+    _iter_levels_meta,
+    _open,
+    _ReadWriteGroup,
+)
```

Comment:
The convention is that underscored names are not exported from the module, so all these names should lose the leading underscore now.

Some people make the module name itself "private" (i.e. `_helpers`). Not too keen on it (most modules are usually "private" from the POV of the package user) but not strongly against it either.

Reply:
Also the `open` function if loses its leading underscore should change name as well as it shadows the built in `open`.

Reply:
Yeap, candidate names off the top of my head: `tiledb_open`, `tdb_open`, `open_level_array`, `open_bioimg_array`, ...

Reply:
How about `open_bioimg`? I like the best `open_bioimg_array `, but I think it is too long so I cropped it. I think that for internal use it is safe to assume that we open an `array`.